documentation/docs/mock-apps/common/verifiable-credentials.md

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_position: 46
+title: Verifiable Credentials
+---
+
+import Disclaimer from '../../\_disclaimer.mdx';
+
+<Disclaimer />
+
+## Description
+
+Verifiable Credentials follow the flow of the W3C Verifiable Credentials Data Model 2.0. It is a standard for issuing, presenting, and verifying credentials in a secure and privacy-preserving manner. Verifiable Credentials are tamper-proof, cryptographically signed documents containing claims about a subject, such as a person, organization, or device. They are issued by trusted entities known as issuers and can be presented to verifiers to prove the authenticity of the claims.
+
+## Credential Securing Mechanism
+
+Verifiable Credentials rely on digital signatures or proofs to ensure the authenticity and integrity of the issued credentials. Two key securing mechanisms are used:
+
+1. **Embedded Proofs**: The proof is included directly within the credential object, typically using JSON Web Signatures (JWS) or JSON-LD Signatures.
+2. **Enveloping Proofs**: The proof wraps the entire credential as a container, using formats like:
+   - **JOSE**: JSON Object Signing and Encryption, which enables signed credentials using JWS.
+   - **COSE**: CBOR Object Signing and Encryption, used for compact and efficient serialization.
+
+Credentials can also be encrypted if sensitive information needs to be protected. Encryption keys are managed through secure Key Management Systems (KMS) to ensure credentials remain confidential and tamper-proof.
+
```

Comment:
Could you provide some specification links for more details about the information in this section, in case someone wants to learn more about it?

Reply:
You can check in here - `Refer to the [Verifiable Credentials Data Model 2.0 - Securing Mechanisms](https://www.w3.org/TR/vc-data-model-2.0/#securing-mechanisms) for more information on the specification.`